README.md

```diff
@@ -1 +1,17 @@
-#pnc-js-libs
\ No newline at end of file
+#pnc-js-libs
+
+library at the moment mainly process pnc notifications
+- pnc-dto-types - dto types generated from swagger json file
+- pnc-messagebus-client - code base for messagebus and notification processing
+- umd-bundle - conversion from typescript to javascript
+
+## Build project
+- npm install
```

Comment:
Even if it may sound obvious, I would mention that following commands should be executed from the root directory, maybe some kind of step by step would be nice:
```sh
git clone <repository> pnc-js-libs
cd pnc-js-libs
npm install 

# generate up-to-date TypeScript type file based on the latest OpenAPI file, output: pnc-dto-types/index.d.ts
# run tests available in pnc-messagebus-client/
# transpile TypeScript to JavaScript, output: umd-bundle/dist/pnc-js-libs.js (it's accessible via bower.json main property)
npm run build
```